### PR TITLE
Integrate sex with db values

### PR DESCRIPTION
**Pull request by ianyechen**

### CODE REVIEW

**Review by angusYuhao — APPROVED** [review]
great stuff

File: widget_pages/patient_form.py [hunk 1]
```diff
@@ -378,9 +377,9 @@ def __init__(self):
         self.setLayout(self.patientFormBigLayout)
 
     def selectedGenderType(self):
-        self.radioButton = self.sender()
-        if self.radioButton.isChecked():
-            print(self.radioButton.gender)
+        gender = self.sender()
```
Comment (liu-allan) [inline widget_pages/patient_form.py:380]:
```suggestion
        sex = self.sender()
```

File: widget_pages/patient_form.py [hunk 2]
```diff
@@ -378,9 +377,9 @@ def __init__(self):
...
+        if gender.isChecked():
```
Comment (liu-allan) [inline widget_pages/patient_form.py:381]: sex

Comment (liu-allan) [inline widget_pages/patient_form.py:381]:
```suggestion
        if sex.isChecked():
```

File: widget_pages/patient_form.py [hunk 3]
```diff
@@ -378,9 +377,9 @@ def __init__(self):
...
+            self.gender = gender.text()
```
Comment (liu-allan) [inline widget_pages/patient_form.py:382]:
```suggestion
            self.sex= sex.text()
```

File: widget_pages/patient_form.py [hunk 4]
```diff
@@ -487,6 +486,7 @@ def savePatientInformation(self):
             ancMeasurement = float(self.ancMeasurementEdit.text())
             dosageMeasurement = float(self.dosageEdit.text())
             age = str(self.calculateAge())
+            sex = self.gender
```
Comment (liu-allan) [inline widget_pages/patient_form.py:489]:
```suggestion
            sex = self.sex
```

File: widget_pages/patient_form.py [hunk 5]
```diff
@@ -614,4 +616,8 @@ def valueChangedDosage(self):
 
     def updatePatientInfo(self):
         self.patient = self.parent().parent().selected_patient
+        # resetting fields 
+        self.gender = "Male"
```
Comment (liu-allan) [inline widget_pages/patient_form.py:620]:
```suggestion
        self.gender = "Male"
```
```suggestion
        self.sex= "Male"
```

Comment (liu-allan) [inline widget_pages/patient_form.py:620]:
```suggestion
        self.sex= "Male"
```

**Review by liu-allan — APPROVED** [review]

**Pull request merged**